### PR TITLE
Create afk-system.md

### PR DESCRIPTION
**Pull request by hfgd123**
## Type of PR

- [X] Translation update / creation
- [ ] Original file creation
- [ ] Original file content (not typo) update
- [ ] Original file typo update

## Checks

If you have changed the content (/ meaning) of an original file, please confirm:

- [N/A] Yes, I have copied the updated section to all translations of this file manually.

## Notes

Please enter additional information here.

### CODE REVIEW

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# AFK-System
+
+Erlaube Nutzern einen AFK-Status zu setzen der anderen Nutzern automatisch mitgeteilt wird, wenn sie versuchen ihn/sie zu erreichen.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md:3]:
```suggestion
Erlaube Nutzern einen AFK-Status zu setzen, der anderen Nutzern automatisch mitgeteilt wird, wenn sie versuchen ihn/sie zu erreichen.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+<ModuleOverview moduleName="afk-system" />
+
+## Funktionen {#features}
+
+* Nutzer können mit [`/afk start`](#commands) eine AFK-Sitzung starten und dabei einen (optionalen) Grund dafür angeben.
+* Wenn ein Nutzer ein Mitglied, das aktuell AFK ist, erwähnt, wird der Bot dem Nutzer mitteilen, dass der Erwähnte Nutzer nicht verfügbar ist. 
+* AFK-Sitzungen werden automatisch beendet, wenn der AFK Nutzer eine Nachricht in einen beliebigen Kanal auf deinem Server sendet (sofern sie dies nicht zu beginn der AFK-Sitzung deaktiviert haben).
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md:11]:
```suggestion
```suggestion
* AFK-Sitzungen werden automatisch beendet, wenn der AFK Nutzer eine Nachricht in einen beliebigen Kanal auf deinem Server sendet (sofern sie dies nicht zu beginn der AFK-Sitzung deaktiviert haben).
```
* AFK-Sitzungen werden automatisch beendet, wenn der AFK Nutzer eine Nachricht in einen beliebigen Kanal auf deinem Server sendet (sofern sie dies nicht zu beginn der AFK-Sitzung deaktiviert haben).
```
```suggestion
* AFK-Sitzungen werden automatisch beendet, wenn der AFK Nutzer eine Nachricht in einen beliebigen Kanal auf deinem Server sendet (sofern sie dies nicht zu Beginn der AFK-Sitzung deaktiviert haben).
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+* AFK-Sitzungen können manuell mit [`/afk end`](#commands) beendet werden.
+* Während einer AFK-Sitzung, wird der Bot dem Namen des Nutzers ein `[AFK]` als Präfix hinzufügen.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md:13]:
```suggestion
* Während einer AFK-Sitzung fügt der Bot zum Namen des Nutzers ein `[AFK]` als Präfix hinzu.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+* AFK-Informationsnachrichten [können angepasst werden](#configuration).
+
+## Einrichtung {#setup}
+
+1. Stelle sicher, dass der Bot die Berechtigungen "Kanal anzeigen", "Nachrichtenverlauf anzeigen" und "Nachrichten senden" in jedem Kanal hat, in dem er auf Erwähnungen von Nutzern, die AFK sind reagieren soll.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md:18]:
```suggestion
1. Stelle sicher, dass der Bot die Berechtigungen "Kanal anzeigen", "Nachrichtenverlauf anzeigen" und "Nachrichten senden" in jedem Kanal hat, in welchen er auf Erwähnungen von Nutzern, die AFK sind, reagieren soll.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+2. Stelle sicher, dass der Bot die Berechtigung hat die Nicknames anderer Nutzer auf deinem Server zu verwalten.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md:19]:
```suggestion
2. Stelle sicher, dass der Bot die Berechtigung hat, die Nicknamen anderer Nutzer auf deinem Server zu verwalten.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+| AFK-Sitzung erfolgreich beendet   | Diese Nachricht wird einem Nutzer angezeigt, wenn er seine AFK-Sitzung mit [`/afk end`](#commands) beendet.                                                                                      |
+| AFK-Sitzung erfolgreich gestartet | Diese Nachricht wird einem Nutzer angezeigt, wenn er seine AFK-Sitzung mit [`/afk start`](#commands) startet.                                                                                  |
+| Nutzer ist mit Begründung AFK          | Diese Nachricht wird gesendet, wenn ein Nutzer erwähnt wird, der aktuell AFK ist und zu Beginn der AFK-Sitzung eine Begründung angegeben hat.                                  |
+| Nutzer ist ohne Begründung AFK       | Diese Nachricht wird gesendet, wenn ein Nutzer erwähnt wird, der aktuell AFK ist und zu Beginn der AFK-Sitzung keine Begründung angegeben hat.                              |
+| AFK-Sitzung automatisch beendet  | Diese Nachricht wird gesendet, wenn ein Nutzer, der aktuell AFK ist und das Automatische Ende der AFK-Sitzung nicht deaktiviert hat, eine Nachricht auf deinem Server schreibt. Dies beendet die AFK-Sitzung |
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md:53]:
```suggestion
| AFK-Sitzung automatisch beendet  | Diese Nachricht wird gesendet, wenn ein Nutzer, der aktuell AFK ist und das Automatische Ende der AFK-Sitzung nicht deaktiviert hat, eine Nachricht auf deinem Server schreibt. Dies beendet die AFK-Sitzung. |
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+## Fehlerbehebung {#troubleshooting}
+
+* Wenn der Bot deinen Nickname nicht ändert, stelle sicher, dass du nicht der Eigentümer des Servers bist. Der Bot kann den Nickname des Eigentümers nicht bearbeiten, selbst wenn er eine höhere Rolle als der Eigentümer hat. Dies ist eine Beschränkung durch Discord.
+* Wenn der Bot die AFK-Nachricht nicht sendet, stelle sicher, dass der Nutzer eine AFK-Sitzung gestartet hat und diese noch nicht beendet wurde.
+* Wenn der Bot eine AFK-Sitzung nicht automatisch beendet, stelle sicher, dass `auto-end` nicht zu Beginn der AFK-Sitzung deaktiviert wurde
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md:59]:
```suggestion
* Wenn der Bot eine AFK-Sitzung nicht automatisch beendet, stelle sicher, dass `auto-end` nicht zu Beginn der AFK-Sitzung deaktiviert wurde.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+## Gespeicherte Daten {#data-usage}
+
+Über alle Nutzer mit laufender AFK-Sitzung werden die folgenden Daten gespeichert:
+
+* Ihre eindeutige Discord User-ID.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md:65]:
```suggestion
* Ihre eindeutige Discord Nutzer-Indentifikationsnummer.
```

**Review by SCDerox — CHANGES_REQUESTED** [review]

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md (hunk 5 above)
Comment (hfgd123) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md:19]: Discord officially calls it "Nicknames" even in German 🤔

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md (hunk 2 above)
Comment (hfgd123) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md:11]: I don't really understand this suggestion

Comment (SCDerox) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md:11]: sry i my brain is working overtime

**Review by Fridolin012 — CHANGES_REQUESTED** [review]

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md (hunk 5 above)
Comment (SCDerox) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md:19]: Wait actually?

Comment (hfgd123) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/tools/afk-system.md:19]: Screenshot_20240320-180533

**Review by Fridolin012 — APPROVED** [review]

**Comment by hfgd123** [comment]
🤔

**Comment by SCDerox** [comment]
Thanks for your contribution - you have received a payout as part of our Open-Source Developer Program. You can find details in your SCNX organization payout history. To pay out the received amount, please follow instructions in your dashboard. If you have any questions, please contact ScootKit staff on scnx.app/help.
Cheers ✌️

**Pull request merged**